src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -33,14 +26,964 @@ to make sure IE does not go into compatibility mode.
   level="warning"
 %}
 
-## Integrations
+&nbsp;
+## Integrating the SDK
+All our JavaScript-related SDKs provide the same API. Still, there are some differences between them, such as installation, which this section of the docs explains.
 
-All of our SDKs provide _Integrations_, which can be seen as some kind of plugins. All JavaScript SDKs provide default _Integrations_; please check details of a specific SDK to see which _Integrations_ it provides.
+**[Mimi note: Drop down here]**
 
-One thing is the same across all our JavaScript SDKs and that's how you add or remove _Integrations_, e.g.: for `@sentry/browser`.
+**[Mimi note: Dropdown which affects install instructions are shown (if nothing is provided in the URL, defaults to browser)]**
 
-### Adding an Integration
+- In the browser
+- Angular
+- Electron
+    - Includes some minidump explanation, maybe redirects to the native page
+- Ember
+- Node.js
+- React
+    - Includes a note pointing to React Native
+- Vue
+- Cordova
+- Ionic
+
+The quickest way to get started is to use the CDN hosted version of the JavaScript browser SDK:
+
+```
+<script src="https://browser.sentry-cdn.com/4.5.3/bundle.min.js" crossorigin="anonymous"></script>
+
+```
+
+&nbsp;
+### Configuring the SDK with your Data Source Name
```

Comment:
what if we just called this section like "Connecting the SDK to Sentry". DSN is a somewhat technical concept that I dont know many people are familiar with.